python/eos/tasks.py

```diff
@@ -512,13 +513,19 @@ def corner_plot(analysis_file:str, posterior:str, base_directory:str='./'):
             xmax = _np.max(xsamples)
             ymin = _np.min(ysamples)
             ymax = _np.max(ysamples)
-            #ax.set_aspect(_np.diff((xmin, xmax))[0] / _np.diff((ymin, ymax))[0])
             ax.hist2d(xsamples, ysamples, weights=f.weights, alpha=1.0, bins=100, cmap='Greys')
             ax.set_xlim((xmin, xmax))
             ax.set_ylim((ymin, ymax))
+            ax.set_aspect(_np.diff((xmin, xmax))[0] / _np.diff((ymin, ymax))[0])
 
     fig.tight_layout()
-    fig.savefig(os.path.join(base_directory, posterior, 'plots', 'corner-plot.png'))
+
+    # This could further be modified with a list of allowed output formats
```

Comment:
I would prefer ``format`` over ``file_extensions``.
Also, I think this is not very clean. I think the pythonic way to handle this is:
```py
_format = copy(format)
if isinstance(_format, str):
    _format = [ _format ]
for f in _format:
    fig.savefig(os.path.join(base_directory, posterior, 'plots', f'corner-plot.{f}'))
```
Copying from the argument is absolutely needed to avoid changing the default value.